bench-vortex/src/bin/tpch_benchmark.rs

```diff
@@ -18,7 +20,7 @@ async fn main() {
     // The formats to run against (vs the baseline)
     let formats = [
         Format::Arrow,
-        Format::Csv,
+        // Format::Csv,
```

Comment:
CSV is so slow that it's not really worth using as a basis of comparison.

once we have fileformat to test against, we'll use parquet as a comparison